resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -250,11 +250,31 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  // the definition of a label in DNS (RFC 1123).
+  private val dns1123LabelFmt = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
+
+  private val podConfValidator = (s"^$dns1123LabelFmt(\\.$dns1123LabelFmt)*$$").r.pattern
+
+  // The possible longest executor name would be "$prefix-exec-${Long.MaxValue}"
+  private def isValidExecutorPodNamePrefix(prefix: String): Boolean = {
+    val reservedLen = Long.MaxValue.toString.length + 6
```

Comment:
`6` might looks a magic number still. could you add some comment here too?

Reply:
updated